docs/core/compatibility/aspnetcore.md

```diff
@@ -8,9 +8,15 @@ ms.author: scaddie
 ---
 # ASP.NET Core breaking changes
 
-ASP.NET Core provides the web app development features used by .NET Core.
+ASP.NET Core provides the web app development features used by .NET Core. 
 
-The following breaking changes are documented on this page:
+Slect one of the following links for breaking changes in a specific version:
+
+* [ASP.NET 5.0](#aspnet-core-50)
```

Comment:
The "Core" suffix is still existing in ASP.NET product name

See https://github.com/dotnet/docs/issues/19979

```suggestion
* [ASP.NET Core 5.0](#aspnet-core-50)
```